src/engine/tools/update_expected.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
```

Comment:
#!/usr/bin/env python3
```suggestion
#!/usr/bin/env python3

from pathlib import Path
```